model.py

```diff
@@ -8029,6 +8040,8 @@ def add_entry(self, edition, annotation=None, first_appearance=None):
             entry.annotation = unicode(annotation)
         if edition.license_pool and not entry.license_pool:
             entry.license_pool = edition.license_pool
+        if featured:
+            entry.featured = True
```

Comment:
I think you should set entry.featured to False if featured is False, since this can be used to edit an existing list entry.